src/librbd/internal.h

```diff
@@ -86,9 +87,13 @@ namespace librbd {
   bool image_options_is_empty(rbd_image_options_t opts);
 
   int snap_set(ImageCtx *ictx, const char *snap_name);
+  int list_images_v2(librados::IoCtx& io_ctx,
```

Comment:
Another leftover of the first implementation. I was using it in ImageDeleter.cc. Not anymore.